sys-process/latencytop/latencytop-0.5-r1.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
```

Comment:
Unfortunately this ebuild belongs to base-system team, and they (as general policy) dont accept EAPI=6 yet, only 5. So let's use 5 instead...